zebra/rt_netlink.c

```diff
@@ -1281,6 +1288,7 @@ static void _netlink_route_build_multipath(const char *routedesc, int bytelen,
 	mpls_lse_t out_lse[MPLS_MAX_LABELS];
 	char label_buf[256];
 	int num_labels = 0;
+	struct vrf *vrf = NULL;
```

Comment:
this is already initialized by vrf_lookup_by_id(), do we need to initialize to NULL here?

Reply:
You're right, it doesn't need to be initialized to NULL. Fixed.